docs/getting-started.md

```diff
@@ -50,20 +62,28 @@ $ cd docs/getting-started
 First of all, you need to define how the data will be organized (a.k.a database schema) in the application with Scalar DB database schema.
 Here is a database schema for the sample application. For the supported data types, please see [this doc](schema.md) for more details.
 
-```sql
-REPLICATION FACTOR 1;
-
-CREATE NAMESPACE emoney;
-
-CREATE TABLE emoney.account (
-    id TEXT PARTITIONKEY,
-    balance INT,
-);
+```json
+{
+  "emoney.account": {
+    "transaction": false,
+    "partition-key": [
+      "id"
+    ],
+    "clustering-key": [],
+    "columns": {
+      "id": "TEXT",
+      "balance": "INT"
+    }
+  }
+}
 ```
 
 To load the schema file, please run the following command.
```

Comment:
```suggestion
Then, download the schema loader that matches with the version you use from [scalardb releases](https://github.com/scalar-labs/scalardb/releases), and run the following command to load the schema.
```

Reply:
Thank you for the suggestion.
Document updated with the above suggestion.